### DISCUSSION TITLE
Column output

### DISCUSSION THREAD

**Discussion by bornsilly** [beetbox/beets#5554] [Q&A]
By lack of a better description, i am wondering if it is possible to have beet output what you query in a column style output so that for example if i say that artist is 40 long it shows 40 characters whether the artist name 1 character long or 50. This way you will see all the fields nicely padded out rather than all being trimmed.

Sor for example:

beet ls -f "$artist - $title ($year) [$genre]"

```
Alison Moyet - Love Resurrection (1984) [Pop]
Alison Moyet - That Ole Devil Called Love (1985) [Pop]
```

becomes:

```
Alison Moyet                    - Love Resurrection                 (1984)  [Pop]
Alison Moyet                    - That Ole Devil Called Love        (1985)  [Pop]
```

I could picture something like:

beet ls -f "$artist{40} $title{50} $year{4} [$genre]{10}" to be a awsome feature if it already doesnt excists.

**Comment by loop333** [comment]
for linux:
.zshrc:
`beet_fa() {
  (echo "fmt^id^year^genre^label^catno^#d^#t^albumartist - album";
  beet ls -a -f '$format^$id^$year^%left{$genre,30}^$label^$catalognum^$disctotal^$albumtotal^$albumartist - $album' path+ "$@") | column -t -s "^"
}
`
